configuration/backend_configuration/globalproperties/i18n-core_demo.xml

```diff
@@ -2,7 +2,7 @@
     <globalProperties>
         <globalProperty>
             <property>locale.allowed.list</property>
-            <value>en, en_GB, es, fr, he, km, ar, vi, pt, zh</value>
+            <value>en, en_GB, es, fr, he, km, ar, vi, pt, zh, in, ro, zh_CN, ka, it</value>
```

Comment:
```suggestion
            <value>en, en_GB, es, fr, he, km, ar, vi, pt, zh, id, ro, zh_CN, ka, it</value>
```

The standard ISO code for Indonesian is `id`, not `in` https://www.iso.org/obp/ui/#iso:code:3166:ID.